docs/guides/server/keycloak-truststore.adoc

```diff
@@ -2,20 +2,35 @@
 <#import "/templates/kc.adoc" as kc>
 
 <@tmpl.guide
-title="Configuring trusted certificates for outgoing requests"
-summary="How to configure the {project_name} Truststore to communicate with external services through TLS."
+title="Configuring trusted certificates"
+summary="How to configure the {project_name} Truststore to communicate through TLS."
 includedOptions="">
 
-When {project_name} communicates with external services through TLS, it has to validate the remote server’s certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks. The certificates of these remote server’s or the CA that signed these certificates must be put in a truststore. This truststore is managed by the Keycloak server.
+When {project_name} communicates with external services or has an incoming connection through TLS, it has to validate the remote certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks.
 
-The truststore is used when connecting securely to identity brokers, LDAP identity providers, when sending emails, and for backchannel communication with client applications. It is also useful
-when you want to change the policy on how host names are verified and trusted by the server.
+The certificates of these clients or servers, or the CA that signed these certificates, must be put in a truststore. This truststore is then configured for use by Keycloak.
 
-By default, a truststore provider is not configured, and any TLS/HTTPS connections fall back to standard Java Truststore configuration. If there is no trust established, then these outgoing requests will fail.
+== Configuring the Global Truststore
 
-== Configuring the {project_name} Truststore
+You may use the --truststore-paths option to create a single merged truststore, which will include the existing Java default truststore certs. That merged truststore will be used as the system default truststore via the javax.net.ssl properties, and as the default spi-truststore.  For example:
 
-You can add your truststore configuration by entering this command:
+<@kc.start parameters="--truststore-paths=myTrustStore.pfx,myOtherTrustStore.pem" />
+
+The truststores must be PEM files, or use the .pfx or .p12 extension and be in PKCS12 format.  If in PKCS12, the certs must be unencrypted - meaning no password is expected.  
+
+It is still possible to directly set your own javax.net.ssl truststore System properties, but it's recommended to use the --truststore-paths instead.
```

Comment:
What does this mean?

Reply:
Not sure how to improve this.

First paragraph specifies the supported formats. Second paragraph states that we still do support setting Java truststore via the system property.